src/runtime/ui.config/navigation/verticalNavigation.ts

```diff
@@ -23,5 +25,13 @@ export default {
     base: 'relative ms-auto inline-block py-0.5 px-2 text-xs rounded-md -me-1 -my-0.5',
     active: 'bg-white dark:bg-gray-900',
     inactive: 'bg-gray-100 dark:bg-gray-800 text-gray-900 dark:text-white group-hover:bg-white dark:group-hover:bg-gray-900'
+  },
+  divider: {
+    border: {
```

Comment:
Why specify the `border` here? It's already the default.

Reply:
Oh I see. I think I was trying add the padding on the `border` property before realizing it needed to be on the `wrapper.base` and then I forgot to remove the `border` property 😅

It is removed now